### PR TITLE
fix(readme): correct broken links, improve getting started steps

### PR DESCRIPTION
**Pull request by seanforyou23**
This PR cleans up the README. Updating old links, adding some additional info around the getting started steps, and improving the syntax highlighting for code examples.

Sorry about the extra noise for trimming trailing whitespace.

### CODE REVIEW

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react-poc/304/artifacts/0/home/circleci/project/doc/index.html)

File: packages/auto-form/yalc.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (riccardo-forina) [inline packages/auto-form/yalc.lock]: This file isn't needed. PR got merged, can you open a new one to remove it please?

Comment (kahboom) [reply packages/auto-form/yalc.lock]: Whoops sorry, I can open a PR to remove it if you'd like @seanforyou23 ?

Comment (kahboom) [reply packages/auto-form/yalc.lock]: https://github.com/syndesisio/syndesis-react-poc/pull/105

**Pull request merged**